rules/falco_rules.yaml

```diff
@@ -2595,6 +2595,14 @@
   priority: CRITICAL
   tags: [process, mitre_execution]
 
+# This rule detects the execution of the docker client inside a container
+- rule: The docker client is executed in a container
+  desc: An event will trigger every time you run a docker client in a container
+  condition: evt.type = execve and evt.dir=< and container.id != host and proc.name in (docker_binaries)
```

Comment:
can you make the changes to 

```
- list: k8s_client_binaries
   items: [docker, kubectl, crictl, etcdctl, calicoctl]
```
`- rule: K8s client tool executed inside a container`
`condition: spawned_process and container and proc.name in (k8s_client_binaries)`

` tags: [container, mitre_execution]`

Reply:
Thanks for the comments, Kaizhe. I didn't realize the macros for spawned process and container. 

I'll modify the pull request and upload the changes.

Reply:
Also, I need some help for adding DCO signoff to a previous commit of this PR... I resolved a code review and forgot to add it. How could I add the DCO to an already existing commit?
(Sorry for the newbie mistake...).

Reply:
No, not a problem, I had this issue before :) take a look this one: https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits

Reply:
Please rebase from `dev` too